mmgen/models/architectures/sngan_proj/generator_discriminator.py

```diff
@@ -162,7 +167,7 @@ def __init__(self,
             with_spectral_norm=with_spectral_norm)
         self.final_act = build_activation_layer(dict(type='Tanh'))
 
-        self.init_weights()
+        self.init_weights(pretrained)
```

Comment:
About the `pretrianed`, you may refer to the [implementation in BigGAN](https://github.com/open-mmlab/mmgeneration/blob/master/mmgen/models/architectures/biggan/generator_discriminator.py#L360). In some cases, users may only want to load the generator from the whole checkpoint.